db/migrate/20170531081508_add_nameto_users.rb

```diff
@@ -0,0 +1,7 @@
+class AddNametoUsers < ActiveRecord::Migration[5.1]
```

Comment:
Just a small thing — this should be `AddNameToUsers` and the file named `add_name_to_users`